src/include/schema.h

```diff
@@ -121,6 +121,23 @@ struct __wt_table {
 	    &S2C(session)->checkpoint_lock, WT_SESSION_LOCKED_CHECKPOINT, op)
 
 /*
+ * WT_WITH_CHECKPOINT_LOCK_NOWAIT --
+ *	Return error if unable to acquire the checkpoint lock,
+ *	perform an operation if lock acquired, drop the lock.
+ */
+#define	WT_WITH_CHECKPOINT_LOCK_NOWAIT(session, ret, op) do {		\
```

Comment:
@sueloverso : I am looking at an alternative approach to use the underlying function __wt_session_get_btree_ckpt() return an EBUSY. I am investigating if it returns EBUSY while doing a checkpoint as expected. This pull request will change accordingly.